Libraries/Components/Pressable/Pressable.js

```diff
@@ -205,20 +222,42 @@ function Pressable(props: Props, forwardedRef): React.Node {
 
   const [pressed, setPressed] = usePressState(testOnly_pressed === true);
 
-  const accessibilityState =
-    disabled != null
-      ? {...props.accessibilityState, disabled}
-      : props.accessibilityState;
+  let _accessibilityState = {
+    busy: ariaBusy ?? accessibilityState?.busy,
+    checked: ariaChecked ?? accessibilityState?.checked,
+    disabled: ariaDisabled ?? accessibilityState?.disabled,
+    expanded: ariaExpanded ?? accessibilityState?.expanded,
+    selected: ariaSelected ?? accessibilityState?.selected,
+  };
+
+  /**
+   * Removing undefined keys from _accessibilityState
```

Comment:
Is this just to retain the existing snapshot? Maybe someone from the RN team can comment about whether this matters. Either we update the snapshot to include the undefined fields, or we could build up the object with conditionals instead?

Reply:
Yes, this is just to retain existing snapshots. It would be great If someone from RN team can verify regarding this. We can update snapshots. Also that will be more efficient.